sdk/include/game-contract-sdk/game_base.hpp

```diff
@@ -140,9 +140,18 @@ class game : public eosio::contract {
 
     using session_table = eosio::multi_index<"session"_n, session_row>;
 
+    struct [[eosio::table("sesfirstact"), eosio::contract("game")]] session_first_action_row {
+        uint64_t ses_id;
+        bool acted; // true if player did first action
```

Comment:
can we add `acted` to the session table?

session is cleaned when the game finishes or `close` is called.

Reply:
yes, we can, but in that case we should to carefully deploy contracts